src/integrations/utils.js

```diff
@@ -51,5 +52,6 @@ export function transformQueryParamsToUrl() {
 
   const queryParams = new URLSearchParams(search);
   const redirect = queryParams.get('redirect');
-  return `${origin}${redirect ?? ''}`;
+  if (!redirect) return null;
```

Comment:
I think I need to update both these functions to ensure they non-redirect query params & URL hashes

Reply:
make this a ternary